Unaccountable-Magpie/WorkforceManagement
# HR should be able to view department details #3

Purpose:
So HR can view  the department details and see the list of employees to each department.  
How it fits in context of the project:
This is a part of the Workforce application for HR.
Specific Feature affected:
When a user selects the department in the Departments page a list of employees that are in that department will show and the heading will also reflect the name of that department.
How to test (Be thorough!):

1. Clone or checkout the branch mr-department-view-list. 
2. CD to bangazon-workforce/WorkforceManagement/BangazonWebApp
3. type Start BangazonWebApp.sln 
4. Select the BangazonApi SQL Database in the Solution Explorer.
5.Run the BangazonAPI SQL file so that the employees I added in Sales will show up.
6. Visual Studio will launch.
7. Select IIS Express to start the application.
8. When the main page launches, select Departments from the Nav bar.
9. Click on and test each of the IT, SALES and  Accounting, when you select the department a page should load showing the employees and the name of the department should display in the header.

May the force be with you.
